GitUI/UserControls/RevisionGrid.cs

```diff
@@ -3024,28 +3024,25 @@ public void UpdateArtificialCommitCount(IReadOnlyList<GitItemStatus> status)
 
         private void UpdateArtificialCommitCount(IReadOnlyList<GitItemStatus> status, GitRevision unstagedRev, GitRevision stagedRev)
         {
-            int staged = status.Count(item => item.IsStaged);
-            int unstaged = status.Count - staged;
-            if (unstagedRev != null)
+            if (status != null)
```

Comment:
please invert to reduce nesting

Reply:
null check was previously in CheckUncommitedChanged(), but should be in UpdateArtificialCommitCount() where the data is used.

OK to keep?

Reply:
@RussKie ?

Reply:
Apologies if I wasn't clear, I asked to invert the `if` to reduce nesting of the code

![image](https://user-images.githubusercontent.com/4403806/40750081-6cf4752c-64a9-11e8-9727-e7a0eade9bc4.png)

so it looks like 

```
if (status == null) 
{
    return;
}

// the rest of the code
```